azurerm/resource_arm_notification_hub_namespace_test.go

```diff
@@ -25,6 +26,31 @@ func TestAccAzureRMNotificationHubNamespace_free(t *testing.T) {
 					testCheckAzureRMNotificationHubNamespaceExists(resourceName),
 				),
 			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: false,
```

Comment:
This shouldn't be false

Reply:
This has to be false, because it will fail since the state file will have `sku_name` and Azure will have the 'sku' structure...

Reply:
You should be able to switch this back to true
```suggestion
				ImportStateVerify: true,
```